pkg/kube/exec.go

```diff
@@ -75,19 +121,22 @@ func ExecWithOptions(kubeCli kubernetes.Interface, options ExecOptions) (string,
 		Container: options.ContainerName,
 		Command:   options.Command,
 		Stdin:     options.Stdin != nil,
-		Stdout:    options.CaptureStdout,
-		Stderr:    options.CaptureStderr,
+		Stdout:    options.Stdout != nil,
+		Stderr:    options.Stderr != nil,
 		TTY:       tty,
 	}, scheme.ParameterCodec)
 
-	config, err := LoadConfig()
-	if err != nil {
-		return "", "", err
-	}
+	errCh := make(chan error, 1)
+	go func() {
+		err := execute("POST", req.URL(), config,
+			options.Stdin,
+			options.Stdout,
+			options.Stderr,
+			tty)
```

Comment:
```suggestion
		err := execute(
			"POST",
			req.URL(),
			config,
			options.Stdin,
			options.Stdout,
			options.Stderr,
			tty,
		)
```
[nit] I like the formatting here - https://github.com/thanos-io/thanos/blob/main/docs/contributing/coding-style-guide.md#only-two-ways-of-formatting-functionsmethods
Improves readability 😄